deployments/kubernetes/install/helm/values.yaml

```diff
@@ -0,0 +1,20 @@
+hub: docker.io/tiswanso
```

Comment:
Is this right?

Reply:
for now, yes... I've built and pushed the images there and that's where folks have been using them from.  I don't think I have access to push anywhere else.  Another proj thing to get working when CI is integrated.